src/main/webapp/app/shared/model/firebase/firebase.model.ts

```diff
@@ -194,6 +210,11 @@ export class Mutation {
   tumors: Tumor[] = [];
   tumors_uuid: string = generateUuid();
 
+  // Germline
+  mutation_specific_penetrance = new MutationSpecificPenetrance();
+  mutation_specific_inheritance_mechanism = new MutationSpecificInheritanceMechanism();
+  mutation_specific_cancer_risk = new CancerRisk();
```

Comment:
I removed all instances of `mutation_specific` besides within the location based on our conversation. This is so the location will display `Mutation Specific...` to curators when looking at history. However, since we are now checking for something like `Cancer Risk` instead of `Mutation Specific Cancer Risk` when parsing the location, so long as that ending remains the same, we can change the name displayed to curators easily while maintaining our old records.